.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,26 @@
+name: Auto Merge to Production and Deploy
+
+on:
+  schedule:
+    # This runs at 12pm every second Thursday
+    - cron: '0 12 * * 4'
```

Comment:
It seems like every Thursday but not every second Thursday. But I would say that it is okay for now. You can improve this later if you want.

Reply:
Yep, I will create a new card to fix it.